Sources/generate-symbol-graph/main.swift

```diff
@@ -202,6 +226,31 @@ func extractDocumentationCommentsForDirectives() throws -> [String : SymbolGraph
         .compactMap { swiftDocCFrameworkSymbolGraph.symbols[$0] }
         .map { (String($0.title.split(separator: ".").last ?? $0.title[...]), $0) }
     
+    let missingDirectiveSymbolNames: [String] = swiftDocCFrameworkSymbolGraph.relationships.compactMap { relationship in
```

Comment:
What scenarios will this help in? It seems like for `@Article` this still would have passed because it isn't included in the list at all, right?